src/icon/icon-exports.js

```diff
@@ -1,31 +1,24 @@
-/*
-Copyright (c) 2018 Uber Technologies, Inc.
-
-This source code is licensed under the MIT license found in the
-LICENSE file in the root directory of this source tree.
-*/
-
 // @flow
-export {default as Alert} from './alert.js';
-export {default as ArrowDown} from './arrow-down.js';
-export {default as ArrowLeft} from './arrow-left.js';
-export {default as ArrowRight} from './arrow-right.js';
-export {default as ArrowUp} from './arrow-up.js';
-export {default as CheckIndeterminate} from './check-indeterminate.js';
-export {default as Check} from './check.js';
-export {default as ChevronLeft} from './chevron-left.js';
-export {default as ChevronRight} from './chevron-right.js';
-export {default as DeleteAlt} from './delete-alt.js';
-export {default as Delete} from './delete.js';
-export {default as Filter} from './filter.js';
-export {default as Grab} from './grab.js';
-export {default as Menu} from './menu.js';
-export {default as Overflow} from './overflow.js';
-export {default as Plus} from './plus.js';
-export {default as Search} from './search.js';
-export {default as Spinner} from './spinner.js';
-export {default as TriangleDown} from './triangle-down.js';
-export {default as TriangleLeft} from './triangle-left.js';
-export {default as TriangleRight} from './triangle-right.js';
-export {default as TriangleUp} from './triangle-up.js';
-export {default as Upload} from './upload.js';
+export {default as Alert} from './alert';
```

Comment:
need to keep the file extension in all imports

Reply:
ah, good catch, thanks Nadiia!